connect/src/routes/tokenBridge/manual.ts

```diff
@@ -168,6 +170,21 @@ export class TokenBridgeRoute<N extends Network>
       destinationTxs: dstTxIds,
     };
   }
+  
+  async resume(tx: TransactionId): Promise<R> {
+    const attestation = await TokenTransfer.getTransferVaa(this.wh, tx.txid);
+    const id = await TokenTransfer.getTransferMessage(this.wh.getChain(tx.chain), tx.txid);
```

Comment:
if this gets `resume`'d before the vaa is signed, does this fail? maybe use the same method as the cctp with `TokenTransfer.from` + a method to produce a receipt

Reply:
it fails and its ok that it does for now

Reply:
just looked at this again and made that change actually 👍
thanks